app/assets/javascripts/app/collections/notifications.js

```diff
@@ -0,0 +1,109 @@
+app.collections.Notifications = Backbone.Collection.extend({
+  model: app.models.Notification,
+  // URL parameter
+  /* eslint-disable camelcase */
+  url: Routes.notifications({per_page: 10, page: 1}),
+  /* eslint-enable camelcase */
+  page: 2,
+  perPage: 5,
+  unreadCount: 0,
+  unreadCountByType: {},
+
+  initialize: function() {
+    this.pollNotifications();
```

Comment:
In combination with the browser notification stuff I wrote a few lines below the new code could be

```javascript
this.fetch();
this.timeout = 300000 // 5 minutes
setTimeout(setInterval(this.pollNotifications.bind(this), this.timeout).bind(this), this.timeout);
Diaspora.BrowserNotification.requestPermission();
```